databuilder/snippets/ehrql.py

```diff
@@ -0,0 +1,45 @@
+# :snippet minimal-ehrql-import-patients
+from databuilder.tables import patients
+
+
+year_of_birth = patients.date_of_birth.year
+# :endsnippet
+
+# :snippet minimal-ehrql-import-dataset
+from databuilder.query_language import Dataset
+
+
+dataset = Dataset()
+# :endsnippet
+
+# :snippet minimal-ehrql-set-population
+dataset.set_population(year_of_birth >= 2000)
```

Comment:
```suggestion
dataset.set_population(year_of_birth <= 2000)
```

We want births in and _before_ 2000, right?

Reply:
Actually, it's the other way around 🤦🏻‍♂️ The logic is correct (it's the same study as we use elsewhere), but the description in `docs/ehrql-tutorial.md` is not. Fixed in 06f684f